modules/c++/math/include/math/Utilities.h

```diff
@@ -61,7 +61,9 @@ inline double square(double val)
  */
 template <typename T> bool isNaN(T value)
 {
-#ifdef HAVE_ISNAN
+#ifdef HAVE_STD_ISNAN
```

Comment:
Don't you need to include `cmath` if you have `HAVE_STD_ISNAN` vs. `math.h` if you have `HAVE_ISNAN`?